src/cloud-api-adaptor/docker/kind_cluster.sh

```diff
@@ -25,9 +26,20 @@ if [ "$1" == "create" ]; then
     sudo sysctl fs.inotify.max_user_watches=524288
     sudo sysctl fs.inotify.max_user_instances=512
 
+    # Create a kind cluster
+    # If CONTAINER_RUNTIME env variable is set to crio, then set KIND_CONFIG_FILE to kind-config-crio.yaml
+    # Else use kind-config.yaml
```

Comment:
nit: Do we still need this comment as the code and echo seems self-explanatory?

Reply:
Fair point.. I'll remove it